projects/packages/my-jetpack/src/products/class-crm.php

```diff
@@ -173,6 +173,19 @@ public static function has_paid_plan_for_product() {
 		return false;
 	}
 
+	/**
+	 * Get the product-slugs of the paid bundles/plans that this product/module is included in.
+	 *
+	 * @return array
+	 */
+	public static function get_paid_bundles_that_include_product() {
```

Comment:
Do we only have this function in CRM since CRM exists outside of our system?

Reply:
Yes, all we can check for CRM is if the site as a Jetpack Complete plan (because CRM is included with Jetpack Complete). That's all we were doing previously, so that's all we are doing here too, so nothing has changed really.